lib/universal-search.js

```diff
@@ -27,34 +27,88 @@ Search.prototype = {
 
   loadIntoWindow: function(win) {
     console.log('loadIntoWindow start');
-    const document = win.document;
 
     // Sets the app global per window.
     win.universalSearch = win.universalSearch || {}
```

Comment:
I was on the fence about pulling this into a method, just to make the loadIntoWindow more uniform, hence, more scannable, but then, it's a single line of code. Thoughts?